install_config/router/default_haproxy_router.adoc

```diff
@@ -1566,6 +1539,31 @@ haproxy_server_bytes_in_total{namespace="default",pod="hello-rc-vkjqx",route="he
 ...
 ----
 
+* To get an HTML file with metrics, delete the following xref:../../architecture/networking/routes.adoc#env-variables[environment variables]
+from the router deployment configuration file:
++
+----
+$ oc edit service router
+
+- name: ROUTER_LISTEN_ADDR
+  value: 0.0.0.0:1936
+- name: ROUTER_METRICS_TYPE
+  value: haproxy
+----
++
+Launch the stats window using the following URL in a browser, where 1936 is the `STATS_PORT` value:
++
+----
+http://admin:<Password>@<router_IP>:1936
+----
++
+To get the router IP, admin name, and password:
++
+----
+oc describe pod <router_pod>
+----
++
+You get a statistics file in HTML format.
```

Comment:
Do the statistics display or do they automatically download? I might say one of these instead:
"The statistics display."
"An HTML file that contains the statistics downloads."

Reply:
@knobunc Can you help with this? (I tried to set up an QuickLab instance, but I cannot access the stats page). What output does the user get when here?